### DISCUSSION TITLE
How to handle bounces/complaints for emails

### DISCUSSION THREAD

**Discussion by g1sbi** [pocketbase/pocketbase#3060] [Q&A] [answered]
I'm using AWS SES as an SMTP server in my Pocketbase. Are there any built in handlers for bounces and complaints? Or do I need to implement them myself?

**Comment by ganigeorgiev** [comment, marked as answer]
> Are there any built in handlers for bounces and complaints?

No. We don't have such integration and this usually could happen long after PocketBase has sent the email to the mail server.

> Or do I need to implement them myself?

How? Could you give an example of a single _"bounces and complaints"_? If it is for example because of a wrong user email or the recipient reached some quota limit for their inbox, I don't see what you can do in this case.

**Reply by g1sbi** [reply]
Understood. I needed it to blacklist a user email in case they keep bouncing, because I'd risk losing my SES access otherwise. I'll just implement it outside of Pocketbase. Thanks for the answer!

**Comment by ChinatuOJ** [comment]
I’m working on a project on noise and using xlur for analysis. During xlur apply, it’s showing me Lur sqlite database is empty. Please how can i correct it.
